control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -185,20 +185,27 @@ type Reconciler struct {
 }
 
 func (r *Reconciler) ReconcileKind(ctx context.Context, cg *kafkainternals.ConsumerGroup) reconciler.Event {
+	logger := logging.FromContext(ctx)
```

Comment:
nit: could we add the name&namespace of the consumergroup, so we can see it directly from the logs?

Reply:
It's already there

Reply:
injected loggers are pre-configured with a few standard keys, see knative/pkg for details